tests/Mutator/ReturnValue/IntegerNegationTest.php

```diff
@@ -29,34 +29,58 @@ public function test_mutator($input, $expected = null): void
     public function provideMutationCases(): array
     {
         return [
-            'It mutates negative int return to positive' => [
+            'It mutates negative -1 int return to positive' => [
```

Comment:
I think this has to be `-2`

Reply:
mutated code should have -1 in this case, fixed